typescript/packages/well-log-viewer/src/SyncLogViewer.stories.tsx

```diff
@@ -556,26 +562,29 @@ const TemplateWithSelection = (args) => {
             <div style={{ flexDirection: "row" }}>
                 <ToggleButton
                     value="check"
-                    selected={showWell1}
+                    selected={showWell1 && !!args.welllogs[0]}
                     onChange={() => {
+                        if (!args.welllogs[0]) assert("!args.welllogs[0]");
                         setShowWell1(!showWell1);
                     }}
                 >
                     Well 1
                 </ToggleButton>
                 <ToggleButton
                     value="check"
-                    selected={showWell2}
+                    selected={showWell2 && !!args.welllogs[1]}
                     onChange={() => {
+                        if (!args.welllogs[1]) alert("!args.welllogs[1]");
```

Comment:
What is the behavior of alert ? Does it open a window ?

Reply:
alert is a browser message box
![image](https://github.com/equinor/webviz-subsurface-components/assets/86053741/1d0e3696-296e-4aa0-890d-419a1922ad33)

I have added some message for button if arg.wellogs array is shoter that the button number

Reply:
It is a part of interactive story (button click responce) so interactive message is quite allowable.